docs/README.md

```diff
@@ -1,6 +1,11 @@
-# Duke
+# Upcycle
 
-{Give product intro here}
+Hi there, 
+
+Welcome to Upcycle - a perfect desktop app dedicate for managing rental businesses. The nature of these businesses
+requires to deal with tons of data; therefore, noting down all information by notes and pen is not ideal for managers.
+It is developed for rental business managers, who can type fast to efficiently keep track all of their customers, items,
+and transactions via Command Line Interface with a single line of command.
 
 Useful links:
 * [User Guide](UserGuide.md)
```

Comment:
This new User() constructor box should end after the constructor returns
![6](https://user-images.githubusercontent.com/71815678/198497554-0abe026a-6716-4bd4-b315-ff2df679edab.png)